usage/installer/src/main/resources/brooklyn/entity/modaclouds/modaclouds.yaml

```diff
@@ -21,8 +22,5 @@ services:
     brooklyn.config:
       modaclouds.dda.ip: $brooklyn:component("dda").attributeWhenReady("host.address")
       modaclouds.dda.port: $brooklyn:component("dda").attributeWhenReady("modaclouds.dda.port")
-      modaclouds.kb.ip: $brooklyn:component("kb").attributeWhenReady("host.address")
-      modaclouds.kb.port: $brooklyn:component("kb").attributeWhenReady("modaclouds.kb.port")
-      modaclouds.kb.dataset.path: $brooklyn:component("kb").attributeWhenReady("modaclouds.kb.path")
```

Comment:
To update the PR just push more commits to your branch and GitHub will autoupdate. 
Then, once this PR is ready to merge you can squash all the commits into one, for simplicity.